notebooks/community/model_evaluation/automl_tabular_regression_model_evaluation.ipynb

```diff
@@ -0,0 +1,1473 @@
+{
```

Comment:
<p>Thank you for adding the image folder. The image is shown very small now at least from <code>ReviewNB</code>. We want it to be still readable. Please make it a better size if it's not readable.</p><p> </p><p>Is this image updated based on the new pipeline or is this from the original notebook? In case of the second case, please update the image.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>

Reply:
<p>done</p>

Reply:
<p>I still see the image as small, the other notebook the image is large. Is this fixed?</p>

Reply:
<p>image is readable to me</p>